docs/ai/llm-spans.md

```diff
@@ -0,0 +1,86 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: LLM Calls
+--->
+
+# Semantic Conventions for LLM requests
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Configuration](#configuration)
+- [LLM Request attributes](#llm-request-attributes)
+- [Events](#events)
+
+<!-- tocstop -->
+
+A request to an LLM is modeled as a span in a trace.
+
+The **span name** SHOULD be set to a low cardinality value representing the request made to an LLM.
+It MAY be a name of the API endpoint for the LLM being called.
```

Comment:
we don't usually put endpoints in the span names. Perhaps we can stay vague and say that it should contain specific operation name (e.g. [`create_chat_completions`](https://platform.openai.com/docs/api-reference/chat/create)).

See also a comment on metric regarding introducing `llm.operation` attribute